### DISCUSSION TITLE
tutorials/intro-strawberry/07-query

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#127] [course feedback]
# tutorials/intro-strawberry/07-query

Create the entry point to your schema to query featured playlists.

https://www.apollographql.com/tutorials/intro-strawberry/07-query

**Comment by akristen** [comment]
Hi y'all! Wanted to point out that you have an author's note to add a definition: "It's also good practice to include self in the arguments of the resolver function. [TODO LINK TO WHY]" Love your component usage + especially how your codeblocks describe removing and adding new snippets of code. Great work!

**Reply by mabuyo** [reply]
Whoops! Thanks for the catch and the kind words @akristen 🙏🏽

**Comment by joselws** [comment]
Hello! Inside the "Learn more: The anatomy of a GraphQL operation" section, there's a paragraph repeated twice. It starts with "This operation syntax begins with..."

**Reply by lizhennessy** [reply]
Thanks for the catch @joselws! We'll get this fixed.

**Comment by bparanj** [comment]
Suggestion: Provide prompts to use for LLMs that can be used to accomplish this step for our own projects.
